frontend/__tests__/components/Common/NewFunctionsLabel.test.tsx

```diff
@@ -0,0 +1,42 @@
+import NewFunctionsLabel from '@src/components/Common/NewFunctionsLabel';
+import { saveVersion } from '@src/context/meta/metaSlice';
+import { setupStore } from '@test/utils/setupStoreUtil';
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+
+const store = setupStore();
+
+describe('NewFunctionsLabel', () => {
+  const setup = (version: string, createVersion: string) => {
+    store.dispatch(saveVersion(version));
+    return render(
+      <Provider store={store}>
```

Comment:
how to compare 1.0.0-alpha、1.0.0-alpha.1、1.0.0-0.3.7、1.0.0-x.7.z.92。

ref to https://semver.org/lang/zh-CN/

Reply:
done